include/ttmlir/Dialect/TTNN/Utils/Utils.h

```diff
@@ -5,10 +5,89 @@
 #ifndef TTMLIR_DIALECT_TTNN_UTILS_UTILS_H
 #define TTMLIR_DIALECT_TTNN_UTILS_UTILS_H
 
+#include <llvm/Support/CommandLine.h>
+
 #include "ttmlir/Dialect/TT/IR/TTOpsTypes.h"
 #include "ttmlir/Dialect/TTNN/IR/TTNNOpsAttrs.h"
 #include "ttmlir/Dialect/TTNN/IR/TTNNOpsTypes.h"
 
+namespace mlir::tt::ttnn {
```

Comment:
Can you create a new file in Utils directory for this instead of merging into this general existing one?

Reply:
Something like PassOptions.h or PassOverrides.h, same comment for cpp.

Reply:
Done.